clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
> Fixes the tracking for return addresses inside system call and context switch traces.

Please state in this first sentence and probably the title too that this is only about the invariant checker, as it took me a while to figure out what this entire PR was about.  When I hear "retaddr tracking" I think of drwrap: though it wouldn't be involved w/ kernel; so then I thought it was some aspect of converting kernel templates.  Setting the context up front would help.

Reply:
Done.